src/components/services/NotificationService.ts

```diff
@@ -186,5 +186,21 @@ export const NotificationService = (props: Props) => {
     'NotificationServices'
   )
 
+  // Make sure the backup banner is always shown on login if needed. We do this
+  // separately in this effect so that we can hide the banner during current
+  // login session if they so choose.
+  useAsyncEffect(
+    async () => {
+      if (!isLightAccountReminder) return
+
+      await writeAccountNotifInfo(account, 'lightAccountReminder', {
+        isBannerHidden: false,
+        isCompleted: false
+      })
```

Comment:
I'm not super happy about writing to disk if it's meant to be ephemeral, but I understand that your notification data structures are kinda linked to disk, so it would be hard to do it any other way. I guess we can just leave it for now.